ci/common.sh

```diff
@@ -177,7 +186,7 @@ GRPC_SERVER_EXAMPLE_VER=20210819
 
 linux_get_dependencies () {
     apt update
-    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev
+    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev xz-utils
```

Comment:
Why do we need `xz-utils`?

Reply:
I need it to install `static-url` from the `.tar.xz` file. ref: https://github.com/apache/apisix/pull/10989/files#diff-67362488684645b13d1314d9d45bdfd21fed3599970720bccd584a079971dff9R80

Reply:
I hope you checked if there is any other alternative way to install static-url? Or install other tool? Or avoid installing it altogether.

Reply:
I will install xz-utils in quic.yml to avoid installing `xz-utils` altogether.